test_cromwell.sh

```diff
@@ -67,37 +66,54 @@ CENTAUR_LOG=${LOG_DIR}/centaur.log
 cd "${RUN_DIR}"
 mkdir -p ${LOG_DIR}
 
-# Build and run cromwell
-if [ ! -d "cromwell" ]; then
-    echo "Cloning Cromwell"
-    git clone https://github.com/broadinstitute/cromwell.git
-else
-    echo "Using existing Cromwell directory"
+if [[ -n ${CROMWELL_BRANCH} ]]; then
+    if [[ -n ${CROMWELL_JAR} ]]; then
+        echo "Do not specify both a branch and a jar"
+        exit 1
+    fi
+
+    if [ ! -d "cromwell" ]; then
+        echo "Cloning Cromwell"
+        git clone https://github.com/broadinstitute/cromwell.git
+    else
+        echo "Using existing Cromwell directory"
+    fi
+
+    cd cromwell
+    git checkout "${CROMWELL_BRANCH}"
+    git pull
+    echo "Building Cromwell"
+    sbt clean assembly >> ../${ASSEMBLY_LOG} 2>&1
+    cd ..
+    CROMWELL_JAR="cromwell/target/scala-2.11/cromwell-*.jar"
 fi
 
-cd cromwell
-git checkout "${CROMWELL_BRANCH}"
-git pull
-echo "Building Cromwell"
-sbt clean assembly >> ../${ASSEMBLY_LOG} 2>&1
-
-java "${CONFIG_STRING}" -jar target/scala-2.11/cromwell-*.jar server >> ../${CROMWELL_LOG} 2>&1 &
+echo "Starting Cromwell, jar is ${CROMWELL_JAR}"
+# Adding quotes on CONFIG_STRING caused bad things to happen. Sorry shellcheck
+java ${CONFIG_STRING} -jar "${CROMWELL_JAR}" server >> "${CROMWELL_LOG}" 2>&1 &
 
 # Build and run centaur
-cd ..
+cd "${RUN_DIR}"
 
-if [ ! -d "centaur" ]; then
+if [[ ! -d "centaur" ]]; then
```

Comment:
No, and it's a good point although beyond the level of handhold-y I originally intended. I'll put this under the "things maybe I'll clean up in the near future"